xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/rightsUI.vm

```diff
@@ -230,9 +226,17 @@ $xwiki.ssfx.use('js/xwiki/table/livetable.css', true)
                 #else
                   <input type="hidden" name="wiki" value="local"/>
                 #end
-                #set ($colsp = $maxlevel + 1)
+                <input type="hidden" name="clsname" value="$clsname" />
```

Comment:
Missing xml escaping?

Reply:
This was a legacy line I moved around. 
From what I see, this variable is defined in the same template a few lines above, and takes only a couple values, so from what I understand xml escaping wouldn't be mandatory.

Better safe than sorry, I added it in https://github.com/xwiki/xwiki-platform/pull/2985/commits/7c9b03580b04364a047affc5b14ccd90f3c0ba34 :+1: